<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacciaProfiloAmico</class>
 <widget class="QWidget" name="InterfacciaProfiloAmico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(113, 113, 113);
}

QPushButton
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0.507177 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 127, 255));
	font: bold 16pt &quot;Kinnari&quot;;
	color: rgb(255, 170, 0);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

QPushButton:hover
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.507177 rgba(255, 170, 0, 255)); 
	color: rgb(0, 0, 0);
}


QPushButton:hover:!pressed
{
	background-color:qlineargradient(spread:pad, x1:0.244, y1:1, x2:0.837, y2:0.227273, stop:0 rgba(255, 255, 127, 255), stop:0.507177 rgba(0, 0, 0, 255));
	color: rgb(255, 170, 0);
}

#ProfiloUtenteLabel
{
	font: bold italic 32pt &quot;Kinnari&quot;;
	color: rgb(255, 85, 0);
	background-color:rgb(113, 113, 113);
}

#DatiPersonaliLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#ContattiLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#LingueLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#EsperienzeLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

QLabel
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color:rgb(57, 57, 57);
}

#CittaTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#NomeTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#CognomeTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#DataNascitaTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#LuogoNascitaTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#IndirizzoTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#ProvinciaTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#TitoloStudioTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#TelefonoTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

#EmailTextLabel
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QTextEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;s
}

QGroupBox
{
	background-color:rgb(57, 57, 57);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

#line2
{
	background-color:rgb(57, 57, 57);
}

#AvvisoLabel
{
	font: bold 12pt &quot;Kinnari&quot;;
	color: rgb(255, 255, 0);
	background-color:rgb(113, 113, 113);

}
</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>821</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ProfiloUtenteLabel">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>0</y>
     <width>450</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Profilo Utente Cercato</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ChiudiButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>670</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Chiudi</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>660</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QLabel" name="TitoloStudioTextLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>119</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ProvinciaTextLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>83</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CognomeTextLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>47</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IndirizzoTextLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>11</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="IndirizzoLabel">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>11</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Indirizzo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataNascitaLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>83</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Nascita:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CittaLabel">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>47</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Citt√†:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LuogoNascitaTextLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>119</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TitoloStudioLabel">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>119</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Titolo Di Studio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="NomeTextLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>11</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DataNascitaTextLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>83</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ProvinciaLabel">
    <property name="geometry">
     <rect>
      <x>338</x>
      <y>83</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Provincia:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CittaTextLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>47</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="NomeLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CognomeLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>47</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cognome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LuogoNascitaLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>119</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Luogo Nascita:</string>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>3</y>
      <width>10</width>
      <height>155</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>325</y>
     <width>270</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="TelefonoTextLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>E-mail:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="EmailTextLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="DatiPersonaliLabel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>100</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dati Personali</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ContattiLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>295</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Contatti</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LingueLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lingue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EsperienzeLabel">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>420</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Esperienze Lavorative</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="LingueTextBrowser">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>450</y>
     <width>260</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="EsperienzeLavorativeTextBrowser">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>450</y>
     <width>260</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AvvisoLabel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>60</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
